README.md

```diff
@@ -1 +1,91 @@
-# Railway-Tracker
\ No newline at end of file
+# Railway Tracker
+
+## About
+
+Brief description of project.
+
+## Folder & Files Structure
+
+`.github/`: Contains the GitHub Actions workflow files for CI/CD.
+
+`dashboard/`: Contains the Streamlit dashboard code.
+
+`images/`: Contains images used in the README [across the project].
+
+`terraform/`: Contains the Terraform files for deploying the cloud services.
+
+`schema/:` Contains the SQL schema for the database.
+
+## Pre-requisites
+
+- Python 3.11
+- pip3
+- Docker
+- AWS CLI
+- Terraform
+
+## Installation & Setup
+
+- What needs to already be installed
+- How to install required libraries
+- Required ENV variables (and format)
+- Required database structure/seeding
+    - `psql --host [database host] [dbname] -f schema.sql`
+
+Create an `.env` file [in this directory] with the following structure:
+
+```sh
+ACCESS_KEY_ID=XXXXXXXXXX
+SECRET_ACCESS_KEY=XXXXXXXXXX
+BUCKET_NAME=XXXXXXXXXX
+DB_USER=XXXXXXXXXX
+DB_PASSWORD=XXXXXXXXXX
+SCHEMA_NAME=XXXXXXXXXX
+DB_HOST=XXXXXXXXXX
+DB_PORT=XXXXXXXXXX
+DB_NAME=XXXXXXXXXX
+```
+
+`DB_HOST` should be a valid URL to a Postgres database server.
+
```

Comment:
Could potentially add more information for other environment variables if necessary (like you've done with 'DB_HOST').